ubuntu_18.04-stable.dockerfile

```diff
@@ -99,6 +99,11 @@ ENV LD_LIBRARY_PATH $HOME/.local/lib:$LD_LIBRARY_PATH
 
 RUN cd $HOME && nuc_data_make
 
-RUN cd pyne/tests \
+# Install OpenMC API
+RUN cd $HOME/opt && git clone https://github.com/openmc-dev/openmc.git
```

Comment:
Please combine the in a single `RUN` line following the pattern of the others.

Reply:
`WORKDIR` should also be set which avoids needing to cd into opt.